bin/check-es-cluster-status.rb

```diff
@@ -145,13 +145,13 @@ def run
       when 'green'
         ok 'Cluster is green'
       when 'yellow'
-        if !options[:alert_status] == 'RED'
+        if ['YELLOW', ''].include? config[:alert_status]
```

Comment:
probably would just be better to do a `|| options[:alert_status].empty?` but because we do restrict the allowed inputs the risk of `include` is mitigated.